file-upload/file-upload-scala-sbt/fileupload-api/src/main/scala/com/example/fileupload/api/FileuploadService.scala

```diff
@@ -0,0 +1,32 @@
+package com.example.fileupload.api
+
+import akka.NotUsed
+import com.lightbend.lagom.scaladsl.api.transport.Method
+import com.lightbend.lagom.scaladsl.api.{ Service, ServiceCall }
+
+trait FileuploadService extends Service {
+
+  /**
+    * Invoke using:
+     <code> curl -X POST -H "Content-Type: text/plain" -d  "hello world" http://localhost:9000/api/echo </code>
+    */
+  def uppercaseEcho(): ServiceCall[String, String]
```

Comment:
Do we need this? Maybe we should just remove it.

Reply:
I want this to demonstrate how both calls are possible once everything is wired: "this file upload thingy is not replacing what lagom is... it's sitting next to what you already have"

Reply:
Fine for me, but I don't think that someone will ever think that one method on the service descriptor would break another.

Reply:
Fair enough.

That will make more sense in the new implementation where we demonstrate two routers living together.